custom_components/mammotion/translations/de.json

```diff
@@ -168,6 +175,9 @@
       },
       "working_speed": {
         "name": "Aufgabengeschwindigkeit"
+      },
+      "cutting_angle": {
+        "name": "Wegwinkel (°)"
```

Comment:
Please remove the angle symbol, this is provided by home assistant, I've updated the branch this PR is now pointing to for this entity to have the degrees symbol

Reply:
Okay. Good to know that 👍